### PR TITLE
Deduplication of content in career dev framework pages

### PR DESCRIPTION
**Pull request by chrispine**
This removes some of the duplication of content between the career dev framework page and the talent review page.

### CODE REVIEW

File: handbook/engineering/career-development/framework.md [hunk 1]
```diff
@@ -2,10 +2,9 @@
 
 Our career development framework is here to help you understand the expectations of your role, and to provide a common language for you and your manager to discuss and plan your career growth. It is also an important part of our larger goal of ensuring everyone is equitably recognized for the impact they have at work, and to reduce bias in promotions and hiring.
 
-## What are these expectations?
+## What are the expectations of my role?
 
-The currently five levels for software engineers in our framework.
-A level is composed of three categories, each with a summary statement and several example behaviors. These categories are:
+There are currently five levels for software engineers described in our framework. A level is composed of three categories, each with a summary statement and several example behaviors. These categories are:
```
Comment (nicksnyder) [inline handbook/engineering/career-development/framework.md:7]: Right now this page (here and in the lvl 6/7 cells) reads like "there are only 5 levels today but we plan to have 7 levels in the future", and I think it should read more like "there are 7 levels" (and it is just a ephemeral state that 2 of them are not documented yet).

File: handbook/engineering/career-development/talent-review-process.md [hunk 2]
```diff
@@ -50,41 +50,11 @@ Finally, we should have a [retrospective](../../retrospectives/index.md) after e
 
 ## Levels
 
-#### Structure of the levels
+We have seven levels for software engineers at Sourcegraph, derived from Option Impact, a startup compensation tool that Sourcegraph and many other companies use to determine their compensation bands.
```
Comment (nicksnyder) [inline handbook/engineering/career-development/talent-review-process.md:53]:
```suggestion
Our [career development framework](framework.md) defines the levels that exist at Sourcegraph. Compensation for each of these levels is derived from Option Impact, a startup compensation tool that Sourcegraph and many other companies use to determine their compensation bands.
```

File: handbook/engineering/career-development/talent-review-process.md [hunk 3]
```diff
@@ -50,41 +50,11 @@ Finally, we should have a [retrospective](../../retrospectives/index.md) after e
...
 
-We have 7 levels for software engineers at Sourcegraph, derived from Option Impact, a startup compensation tool that Sourcegraph and many other companies use to determine their compensation bands. For each of these, we have a corresponding level description.
+Levels 1-5 are [described in our career development framework](framework.md).
```
Comment (nicksnyder) [inline handbook/engineering/career-development/talent-review-process.md:55]:
```suggestion
```

**Review by nicksnyder — APPROVED** [review]
Thanks!

my feedback is non-blocking

File: handbook/engineering/career-development/framework.md [hunk 4]
```diff
@@ -30,6 +33,8 @@ When your manager can make the case that you’ve had at least one quarter of hi
 Promotions from one level to another are considered in our quarterly [talent review](talent-review-process.md).
 An in-band compensation increase (while staying at the same level) can happen at any time, in recognition of exceeding expectations in your current level without having yet met the expectations of the next level.
 
+You can learn more [here](talent-review-process.md#considerations-for-promotion).
```
Comment (felixfbecker) [inline handbook/engineering/career-development/framework.md:36]: Generally, links should not say "here" or "click here", but the content of the link should be descriptive in itself: https://webaccess.berkeley.edu/ask-pecan/click-here

A big reason for this is accessibility, screen readers allow to jump to links in a document/paragraph/section and will announce the content of the link, but if it only says "here", the user can't know where it leads.

Better would be: `To learn more, read our [talent review process](...).`

Will change this on `main` but just wanted to share this tip for the future :)

Comment (chrispine) [reply handbook/engineering/career-development/framework.md:36]: Thanks!

**Pull request merged**